frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -1,26 +1,62 @@
-import { restore } from "__support__/e2e/cypress";
+import { restore, describeEE, describeOSS } from "__support__/e2e/cypress";
 
 describe("scenarios > admin > databases > list", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
   });
 
-  it.skip("should not display error messages upon a failed `GET` (metabase#20471)", () => {
-    const errorMessage = "Lorem ipsum dolor sit amet, consectetur adip";
-
-    cy.intercept("GET", "/api/database", req => {
-      req.reply({
-        statusCode: 500,
-        body: { message: errorMessage },
-      });
-    }).as("failedGet");
-
-    cy.visit("/admin/databases");
+  describeOSS("OSS", () => {
+    it("should not display error messages upon a failed `GET` (metabase#20471)", () => {
+      const errorMessage = "Lorem ipsum dolor sit amet, consectetur adip";
+
+      cy.intercept(
+        {
+          method: "GET",
+          pathname: "/api/database",
+        },
+        req => {
+          req.reply({
+            statusCode: 500,
+            body: { message: errorMessage },
+          });
+        },
+      ).as("failedGet");
+
+      cy.visit("/admin/databases");
+
+      cy.wait("@failedGet");
+      // Not sure how exactly is this going the be fixed, but we should't show the full error message on the page in any case
+      cy.findByText(errorMessage).should("not.exist");
+    });
+  });
 
-    cy.wait("@failedGet");
-    // Not sure how exactly is this going the be fixed, but we should't show the full error message on the page in any case
-    cy.findByText(errorMessage).should("not.exist");
+  describeEE("EE", () => {
+    it("should not display error messages upon a failed `GET` (metabase#20471)", () => {
+      const errorMessage = "Lorem ipsum dolor sit amet, consectetur adip";
+
+      cy.intercept(
+        {
+          method: "GET",
+          pathname: "/api/database",
+          query: {
+            exclude_uneditable_details: "true",
+          },
```

Comment:
Intercept different requests based on whether the version is Enterprise or not. This is needed after https://github.com/metabase/metabase/pull/20953 is merged